### PR TITLE
(maint) Update beaker and install method

### PR DESCRIPTION
**Pull request by melissa**
This commit also changes all uses of puppetlabs.lan to
delivery.puppetlabs.net

### CODE REVIEW

File: acceptance/lib/puppet/acceptance/install_utils.rb [hunk 1]
```diff
@@ -119,44 +119,51 @@ def stop_firewall_on(host)
       def install_repos_on(host, project, sha, repo_configs_dir)
         platform = host['platform'].with_version_codename
         platform_configs_dir = File.join(repo_configs_dir,platform)
-        tld     = sha == 'nightly' ? 'nightlies.puppetlabs.com' : 'builds.puppetlabs.lan'
+        dev_builds_url  = ENV['DEV_BUILDS_URL'] || 'http://builds.delivery.puppetlabs.net'
+        tld     = sha == 'nightly' ? 'http://nightlies.puppetlabs.com' : dev_builds_url
```
Comment (MikaelSmith) [inline acceptance/lib/puppet/acceptance/install_utils.rb]: Any reason to avoid https here?

Comment (melissa) [reply acceptance/lib/puppet/acceptance/install_utils.rb]: Nope, that's just what was being used prior

File: acceptance/lib/puppet/acceptance/install_utils.rb [hunk 2]
```diff
@@ -119,44 +119,51 @@ def stop_firewall_on(host)
       def install_repos_on(host, project, sha, repo_configs_dir)
         platform = host['platform'].with_version_codename
         platform_configs_dir = File.join(repo_configs_dir,platform)
-        tld     = sha == 'nightly' ? 'nightlies.puppetlabs.com' : 'builds.puppetlabs.lan'
+        dev_builds_url  = ENV['DEV_BUILDS_URL'] || 'https://builds.delivery.puppetlabs.net'
+        tld     = sha == 'nightly' ? 'https://nightlies.puppetlabs.com' : dev_builds_url
```
Comment (melissa) [inline acceptance/lib/puppet/acceptance/install_utils.rb]: @MikaelSmith I updated the urls to `https`

**Comment by melissa** [comment]
No mco tests in the ad hoc pipeline, but successful puppet-agent tests https://jenkins-master-prod-1.delivery.puppetlabs.net/view/puppet-agent/view/ad-hoc/job/platform_puppet-agent_intn-van-sys-suite-puppet-agent-daily-ad-hoc/122/

**Comment by MikaelSmith** [comment]
Passed locally for me.

**Comment by melissa** [comment]
@MikaelSmith we might want to revert this. Apparently the puppet tests broke because of these changes

**Comment by MikaelSmith** [comment]
Will leave that up to you, they're not being run at the moment.

**Pull request merged**